### PR TITLE
Fix mpirun.1.rst typos

### PR DESCRIPTION
**Pull request by jsquyres**
Thanks to @satishskamath for raising the issue.

Refs #12936 

@rhc54 Can you confirm the PRRTE / PRTE fixes on this PR?  (restricted to the 1st commit)

### CODE REVIEW

File: docs/man-openmpi/man1/mpirun.1.rst [hunk 1]
```diff
@@ -1013,10 +1013,12 @@ MCA parameters can be set not only on the mpirun command line, but
 alternatively in a system or user ``mca-params.conf`` file or as
 environment variables, as described in the :ref:`Setting MCA
 Parameters <man1-mpirun-setting-mca-parameters>`. These are MCA parameters for
-the PRRTE runtime so the command line argument ``--PRRTEmca`` must be used to
-pass the MCA parameter key/value pair. Alternatively, the MCA parameter key/
-value pair may be specific on the command line by prefixing the key with
-``PRRTE_MCA_``. Some examples include:
+the PRRTE runtime so the command line argument ``--prte-mca``
```
Comment (rhc54) [inline docs/man-openmpi/man1/mpirun.1.rst:1016]: It is `--prtemca` - all one word

File: docs/man-openmpi/man1/mpirun.1.rst [hunk 2]
```diff
@@ -1662,7 +1664,7 @@ that job are designated "secondary" jobs):
   summary print statement.
 
 By default, the job will abort when any process terminates with
-non-zero status. The MCA parameter ``--PRRTEmca state_base_error_non_zero_exit``
+non-zero status. The MCA parameter ``--prte-mca state_base_error_non_zero_exit``
```
Comment (rhc54) [inline docs/man-openmpi/man1/mpirun.1.rst:1667]: Ditto here - it is `--prtemca`

File: docs/man-openmpi/man1/mpirun.1.rst [hunk 3]
```diff
@@ -532,11 +532,11 @@ generally useful to most Open MPI users:
   just before launch.
 
 * ``--launch-agent``: Name of the executable that is to be used to
-  start processes on the remote nodes. The default is ``PRRTEd``. This
+  start processes on the remote nodes. The default is ``prted``. This
   option can be used to test new daemon concepts, or to pass options
   back to the daemons without having mpirun itself see them. For
-  example, specifying a launch agent of ``PRRTEd -mca odls_base_verbose
-  5`` allows the developer to ask the ``PRRTEd`` for debugging output
+  example, specifying a launch agent of ``prted --mca odls_base_verbose
```
Comment (rhc54) [inline docs/man-openmpi/man1/mpirun.1.rst:538]: Your example isn't quite correct here - it should be `--prtemca odls_base_verbose 5`.  Using the generic `mca` leaves open the possibility that `prted` will misinterpret the intent.

**Review by rhc54 — CHANGES_REQUESTED** [review]
Few minor corrections. Do you also cover PMIx params in here somewhere? I didn't see them mentioned.

**Comment by jsquyres** [comment]
Thanks @rhc54 -- fixed.

**Comment by jsquyres** [comment]
Hey @rhc54 -- I added blurbs about `--pmixmca` and `--prtemca`.  Do those look ok?

**Comment by rhc54** [comment]
> Hey @rhc54 -- I added blurbs about `--pmixmca` and `--prtemca`. Do those look ok?

Definitely!! Thanks!

**Pull request merged**